### ISSUE TITLE
Plans of releasing training codes for Maze and Block stacking dataset

### ISSUE THREAD

**Issue by xmlyqing00** [jannerm/diffuser#5]
Hi,

I'm really interested in your work. It's awesome. Do you have any plans to release training/testing codes and configs for Maze the Block stacking datasets? It would be a great help to understand and follow this paper better.

Thanks.

**Comment by yilundu** [comment]
Yes, we will release the code for the remaining datasets soon.

**Comment by xmlyqing00** [comment]
Sounds perfect! My summer project counts on it HAHA. I'm trying to implement your model for Maze task today because of the short time schedule. I leave this issue open so I can get a notice when your code is released.

**Comment by yilundu** [comment]
Hi,

The code for Maze and Block stacking datasets should be released now -- we are working on releasing the gradient guided value function sampling soon.

**Comment by Cranial-XIX** [comment]
Thank you very much! Looking forward to your sampling code!

Best,
Bo

**Comment by jannerm** [comment]
The sampling code has now been released; see [this section of the README](https://github.com/jannerm/diffuser#using-pretrained-models) for instructions on how to run it. Let us know if you run into any issues!
